internal/protocol/completion.go

```diff
@@ -0,0 +1,9 @@
+package protocol
+
+import "github.com/hashicorp/hcl-lang/lang"
+
+type CompletionItemR struct {
```

Comment:
```suggestion
type CompletionItemWithResolveHook struct {
```
Just for some extra clarity